README.md

```diff
@@ -156,31 +156,39 @@ staramr db restore-default
 
 ## Bioconda
 
+
+### Separate conda environment
+
 The easiest way to install `staramr` is through [Bioconda][bioconda] (we recommend using [mamba](https://mamba.readthedocs.io/) as an alternative to `conda`).
 
 ```bash
 conda install mamba # Install mamba to make it easier to install later dependencies
-mamba install -c bioconda -c conda-forge -c defaults staramr==0.7.2 pandas==1.1.5 mlst==2.19.0
+mamba create -c conda-forge -c bioconda -c defaults --name staramr staramr
 ```
 
-This will install the `staramr` Python package at version `0.7.2` (replace with whichever version you wish to install). Bioconda will install all necessary dependencies and databases (use `pandas==1.1.5` and `mlst==2.19.0` to solve issues installing the correct dependency versions). Once this is complete you can run:
+This will install the `staramr` software at the most recent version within the conda environment named `staramr`. Bioconda will install all necessary dependencies and databases. Once this is complete you can run:
 
 ```bash
+conda activte staramr # Activate conda environment
 staramr --help
 ```
 
-If you wish to use `staramr` in an isolated environment (in case dependencies conflict) you may alternatively install with:
+### Same conda environment
+
+If, instead, you wish to install `staramr` to the current conda environment you can run:
 
 ```bash
-conda install mamba # Install mamba to make it easier to install later dependencies
-mamba create -c bioconda -c conda-forge -c defaults --name staramr staramr==0.7.2 pandas==1.1.5 mlst==2.19.0
+mamba install -c conda-forge -c bioconda -c defaults staramr
 ```
 
-To run `staramr` in this case, you must first activate the environment.  That is:
+You should now be able to run `staramr --help` and recieve a usage statement.
+
+### Using Python 3.7
+
+If you wish to install staramr within a conda environment with Python 3.7, you will have to include `'numpy <1.23.5'` as numpy >= 1.23.5 is [not compatible with Python 3.7](https://github.com/numpy/numpy/releases/tag/v1.23.5).
```

Comment:
As per already discussed, if Python 3.7 is the only option and this is already done for the user, then this might not be necessary.

Reply:
Thanks. Removed this in f41df9dbe2803fe7ab3808a4e01c714842653cd4